controllers/story.js

```diff
@@ -86,10 +86,12 @@ exports.recent = function(req, res) {
 exports.preSubmit = function(req, res) {
 
     var data = req.query;
-    var cleanData = sanitizeHtml(data.url);
+    var cleanData = sanitizeHtml(data.url, {
+                      allowedTags: [],
+                      allowedAttributes: []
+                    }).replace(/&quot;;/g, '"');
```

Comment:
This is inconsistent with [line 231](https://github.com/FreeCodeCamp/freecodecamp/pull/190/files#diff-073009843fcbce1ddd72f81c8a2034f0R235) as well as the rest of the site. 

Vertical Space is cheap, while horizontal space is costly.